core/fixnum/coerce_spec.rb

```diff
@@ -1,39 +0,0 @@
-require File.expand_path('../../../spec_helper', __FILE__)
-
-describe "Fixnum#coerce when given a Fixnum" do
```

Comment:
There is actually Integer#coerce though:
```ruby
[1] pry(main)> 3.method :coerce
=> #<Method: Integer#coerce>
```
So I think this should be moved to `core/integer/coerce_spec.rb`